core/serial_command.py

```diff
@@ -81,12 +81,18 @@ class SerialCommand(object):
     """Contains a serial command object.
     """
 
-    def __init__(self, port_name=None):
+    def __init__(self, port_name=None, port_timeout=None):
         """Initialise the serial command"""
         if not port_name:
-            self._port_name = "/dev/tty.usbserial-FTGA2OCZ"
+            self._port_name = "/dev/ttyUSB0"
         else:
             self.logger = tellie_logger.TellieLogger.get_instance()
+            self._port_name = port_name
+            
+        if not port_timeout:
```

Comment:
IIRC we did the fix in the lab (Mark, James, myself), and it worked with 'None'. Any reason to believe it's different for 0?